### PR TITLE
fix create profile issues

### LINKED ISSUE #15544: Name initials are not dynamically updated within profile icon according to display name value entered on Create profile screen (Android)
**Steps:** 

1. Install app
2. Proceed with onboarding (get to the Create profile screen)
3. Enter name value
4. See if name initials within profile icon are updated according to the entered name value

**Expected result:** name initials are being updated dynamically

**Actual result:** name initials are not being updated dynamically (only after backgrounding of the app). Also, name initials are being dislocated to the right when user starts entering name value. See video below. 

https://user-images.githubusercontent.com/97245802/229051480-4c006fb9-7919-46fc-9090-a000d8a644a5.mp4

### Additional Information

- Status version: nightly 
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)

### LINKED ISSUE #15553: Profile icon disappears after setting up profile picture (Android)
**Case 1:** 

**Steps:** 

1. Install app
2. Proceed to Create profile onboarding step
3. Set profile picture
4. See if profile icon is displayed 

**Actual result:** profile icon disappeared. It appears again after re-opening Create profile screen. 

5. Finish onboarding and enter multiaccount
6. Perform logout
7. See if profile icon is displayed on login page 

**Actual result:** profile icon disappeared. It appears again after re-opening the app. It will dissapear after every logout.   

https://user-images.githubusercontent.com/97245802/229128706-151d4688-a294-4cd3-bb10-d2436e72cafb.mp4

**Case 2:**

**Steps:** 

1. Create/restore multiaccount. Do not set profile picture during onboarding
2. Set profile picture after entering multiaccount
3. Perform logout
4. See if profile icon is displayed on login page

**Actual result:** profile icon disappeared. It appears again after re-opening the app. It will dissapear after every logout.   

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/15544 
fixes https://github.com/status-im/status-mobile/issues/15553

status: ready

### CODE REVIEW

File: src/quo2/components/inputs/profile_input/view.cljs [hunk 1]
```diff
@@ -18,20 +18,14 @@
     [rn/view
      {:style (style/container customization-color)}
      [rn/view
-      [hole-view/hole-view
```
Comment (Parveshdhull) [inline src/quo2/components/inputs/profile_input/view.cljs:21]: dynamic updates inside hole-view don't work
Some context: https://github.com/status-im/status-mobile/blob/c5de05432436798a87a205edc91082d8e4f07161/src/quo2/components/navigation/bottom_nav_tab.cljs#L52

**Review by ibrkhalil — APPROVED** [review]

**Comment by Parveshdhull** [comment]
### Testing note:

PR removes hole view which was creating above issues.
So now color will not be visible below camera icon as design (only on android)

#### PR
image

#### Design
image

this will be fixed separately.

**Comment by J-Son89** [comment]
hmm, let's put some time into addressing this properly and use this solution as a fallback if we can not find a better alternative by the end of the day. I understand it's a blocker for release but this issue will be common across the application and in my opinion it is worth spending some time investigating now

**Comment by pavloburykh** [comment]
> hmm, let's put some time into addressing this properly and use this solution as a fallback if we can not find a better alternative by the end of the day. I understand it's a blocker for release but this issue will be common across the application and in my opinion it is worth spending some time investigating now

thanx @J-Son89 @Parveshdhull please, let us now once you will find alternative way to fix the issue and it will be ready for texting. 

Meanwhile I have checked current fix and it works for me.

**Comment by J-Son89** [comment]
> > hmm, let's put some time into addressing this properly and use this solution as a fallback if we can not find a better alternative by the end of the day. I understand it's a blocker for release but this issue will be common across the application and in my opinion it is worth spending some time investigating now
> 
> thanx @J-Son89 @Parveshdhull please, let us now once you will find alternative way to fix the issue and it will be ready for texting.
> 
> Meanwhile I have checked current fix and it works for me.

awesome, thank you @pavloburykh.  :)

**Review by J-Son89 — APPROVED** [review]
looks good to merge. Should this be going to release branch first though 🤔 

Perhaps we should create an issue and include a comment/TODO to here?

**Comment by Parveshdhull** [comment]
Hi @J-Son89, Thank you for reviewing PR.

> Should this be going to release branch first though

https://github.com/status-im/status-mobile/blob/develop/doc/release-guide.md#bugfixes-before-an-app-is-released
`The best workflow is to send a PR to develop, get it merged, and the cherry-pick on the release branch`

> Perhaps we should create an issue and include a comment/TODO to here?

Issue sounds good

**Comment by J-Son89** [comment]
@cammellos  related to this issue - 
It seems the image will update if the user avatar uses react native image rather than fast-image.
@ulisesmac  @ibrkhalil   any idea how important fast-image is to this component or it's fine to use rn/image instead?? 🤔

**Comment by ibrkhalil** [comment]
> @cammellos related to this issue - It seems the image will update if the user avatar uses react native image rather than fast-image. @ulisesmac @ibrkhalil any idea how important fast-image is to this component or it's fine to use rn/image instead?? 🤔

I think for slower devices fast-image is a good choice because of caching and it can support gifs if we ever wanted to make the user have animated photos
More info in it's [readme](https://github.com/DylanVann/react-native-fast-image)

**Pull request merged**